librz/bin/bobj_process_class.c

```diff
@@ -0,0 +1,92 @@
+// SPDX-FileCopyrightText: 2023 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2023 deroad <wargio@libero.it>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+static void process_class_method(RzBinObject *o, RzBinSymbol *method) {
+	// rebase physical address
+	method->paddr += o->opts.loadaddr;
+
+	char *key = rz_str_newf(RZ_BIN_FMT_CLASS_HT_GLUE, method->classname, method->name);
+	if (!key) {
+		RZ_LOG_ERROR("bin: failed to allocate class-method string\n");
+		return;
+	}
+
+	if (!ht_pp_find(o->glue_to_class_method, key, NULL)) {
+		ht_pp_insert(o->glue_to_class_method, key, method);
+	}
```

Comment:
same and everywhere else

Reply:
no, this cannot use the insert directly, the issue is that sometimes the key might exists and we do not want to overwrite dups

Reply:
I will check but I don't think it overwrites. The update function overwrites , the insert returns false if there is already an element

Reply:
when i have used the insert function, it was weird the output.